### PR TITLE
Remove Add File button. Fix permission functionalities. Remove unused code 

### PR DESCRIPTION
**Pull request by JLaRue96**

### CODE REVIEW

File: syncr_frontend/frontend.py [hunk 1]
```diff
@@ -209,7 +210,10 @@ def get_permission(drop_id):
     owned_drops = get_owned_drops()
 
     for drop in owned_drops:
-        if drop['name'] == drop_id:
+
+        curr_id = base64.b64decode(drop.get('drop_id'), altchars=b'+-')
```
Comment (matthewbentley) [inline syncr_frontend/frontend.py]: the frontend probably shouldn't be caring about decoded drop ids

Comment (JLaRue96) [reply syncr_frontend/frontend.py]: should I just change it over to if drop.get('drop_id') == drop_id then?

Comment (matthewbentley) [reply syncr_frontend/frontend.py]: probably

**Comment by JLaRue96** [comment]
#23

**Comment by JLaRue96** [comment]
Also fixes #29

**Comment by JLaRue96** [comment]
I will also do the git commands (like in backend) as soon as I get this merged.

File: syncr_frontend/frontend.py [hunk 2]
```diff
@@ -2,13 +2,13 @@
...
 from os import path
 from os import scandir
-from tkinter import filedialog
-from tkinter import Tk
 
 from flask import flash
 from flask import Flask
 from flask import render_template
 from flask import request
+from tkinter import filedialog
```
Comment (matthewbentley) [inline syncr_frontend/frontend.py]: is this still used?

Comment (JLaRue96) [reply syncr_frontend/frontend.py]: it's used in select_directory...not sure if tryjankowski was using that for his create drop / subscribe to drop stuff.

If not, I could totally just delete select_directory and the TK stuff

Comment (JLaRue96) [reply syncr_frontend/frontend.py]: I'll text him and see if he uses it

**Pull request merged**